samples/apps/copilot-chat-app/webapi/CopilotChat/Options/PlannerOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SemanticKernel.Service.CopilotChat.Options;
+
+/// <summary>
+/// Configuration options for the planner.
+/// </summary>
+public class PlannerOptions
```

Comment:
Not required now, but at some point might be worth adding relevancy filtering if plugin function count gets large.